config/foreman.migrations/20200909151007_manage_acls_on_debian.rb

```diff
@@ -0,0 +1 @@
+answers['foreman_proxy::dhcp_manage_acls'] ||= true
```

Comment:
The structure is a bit different. `answers` is a mapping of class name to the answers of that class. The answers can be `false` (class disabled), `true` (class enabled, but no specific answers`) or a hash, mapping parameter to value.
```suggestion
if answers['foreman_proxy'].is_a?(Hash)
  answers['foreman_proxy']['dhcp_manage_acls'] ||= true
end
```